server/mailet/mock-smtp-server/pom.xml

```diff
@@ -119,4 +119,60 @@
             <version>3.1.7</version>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>1.5.0</version>
+                <configuration>
+                    <from>
+                        <image>openjdk:8u222-jre</image>
+                    </from>
+                    <to>
+                        <image>linagora/mock-smtp-server</image>
+                        <tags>
+                            <tag>latest</tag>
+                        </tags>
+                    </to>
+                    <container>
+                        <mainClass>org.apache.james.mock.smtp.server.MockSMTPServerMain</mainClass>
+                        <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+                        <ports>
+                            <port>25</port>
+                            <port>8000</port>
+                        </ports>
+                    </container>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>docker packaging</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>buildTar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.coderplus.maven.plugins</groupId>
+                <artifactId>copy-rename-maven-plugin</artifactId>
+                <version>1.0</version>
```

Comment:
Use plugin mgt of `/pom.xml` for specifying plugin version (and for JIB too?) in pluginManagement section.

This avoids lurking in all POMs when doing an upgrade for instance